singer-io/tap-tiktok-ads
# Tdl 17935 add custom error handling and backoff

# Description of change

- Added proper error handling based on return codes in the response
- Added retry for the code 50000

# Manual QA steps
 - Check that the tap raises an error with proper error messages in case of error
 - Check that the request is backoff for 50000 error code.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
